Violet 12 exercises py.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Nice! I like your use of an error message when the user puts an invalid call. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>